### PR TITLE
feat: add pause in drag_and_drop

### PR DESCRIPTION
**Pull request by KazuCocoa**
closes https://github.com/appium/python-client/issues/955

### CODE REVIEW

File: appium/webdriver/extensions/action_helpers.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def drag_and_drop(
         Args:
             origin_el: the element to drag
             destination_el: the element to drag to
-            pause: how lon the action pauses before moving after the tap and hold.
+            pause: how lon the action pauses before moving after the tap and hold, in seconds.
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply appium/webdriver/extensions/action_helpers.py:72]: in float seconds

[the remark this answers is not in the document]
Comment (KazuCocoa) [reply appium/webdriver/extensions/action_helpers.py]: updated

[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply appium/webdriver/extensions/action_helpers.py]: `how lon` is still there

[the remark this answers is not in the document]
Comment (KazuCocoa) [reply appium/webdriver/extensions/action_helpers.py]: Maybe an old reference?
Current latest:
Screenshot 2024-02-22 at 12 20 08 PM

File: appium/webdriver/extensions/action_helpers.py [hunk 2]
```diff
@@ -61,19 +61,24 @@ def scroll(self, origin_el: WebElement, destination_el: WebElement, duration: Op
         actions.perform()
         return cast('WebDriver', self)
 
-    def drag_and_drop(self, origin_el: WebElement, destination_el: WebElement) -> 'WebDriver':
+    def drag_and_drop(
+        self, origin_el: WebElement, destination_el: WebElement, pause: Union[float, None] = None
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/action_helpers.py:65]: I prefer Optional[float]

File: appium/webdriver/extensions/action_helpers.py [hunk 3]
```diff
@@ -61,19 +61,24 @@ def scroll(self, origin_el: WebElement, destination_el: WebElement, duration: Op
...
             destination_el: the element to drag to
+            pause: how long the action pauses before moving after the tap and hold, in float seconds.
 
         Returns:
             Union['WebDriver', 'ActionHelpers']: Self instance
         """
         actions = ActionChains(self)
         # 'mouse' pointer action
         actions.w3c_actions.pointer_action.click_and_hold(origin_el)
+        if pause:
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/action_helpers.py:80]: if pause is not None

File: appium/webdriver/extensions/action_helpers.py [hunk 4]
```diff
@@ -61,19 +61,24 @@ def scroll(self, origin_el: WebElement, destination_el: WebElement, duration: Op
...
+            actions.w3c_actions.pointer_action.pause(pause)
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/action_helpers.py:81]: I assume we need to convert the value to integer milliseconds according to w3c spec

Comment (KazuCocoa) [reply appium/webdriver/extensions/action_helpers.py:81]: selenium client takes care it
https://github.com/SeleniumHQ/selenium/blob/93c780439f2886607474a6cebaf4e678f9730047/py/selenium/webdriver/common/actions/pointer_actions.py#L198-L200
https://github.com/SeleniumHQ/selenium/blob/93c780439f2886607474a6cebaf4e678f9730047/py/selenium/webdriver/common/actions/pointer_input.py#L63-L64

**Pull request merged**